mev_inspect/s3_export.py

```diff
@@ -21,39 +21,71 @@
     block_number = :block_number
     """
 
+ARBITRAGES_EXPORT_QUERY = """
+    SELECT to_json(arbitrages)
+    FROM arbitrages
+WHERE
+    block_number = :block_number
+    """
+LIQUIDATIONS_EXPORT_QUERY = """
+    SELECT to_json(liquidations)
+    FROM liquidations
+WHERE
+    block_number = :block_number
+    """
+SANDWICHES_EXPORT_QUERY = """
+    SELECT to_json(sandwiches)
+    FROM sandwiches
+WHERE
+    block_number = :block_number
+    """
+
+query_by_table = {
+    "mev_summary": MEV_SUMMARY_EXPORT_QUERY,
+    "arbitrages": ARBITRAGES_EXPORT_QUERY,
+    "liquidations": LIQUIDATIONS_EXPORT_QUERY,
+    "sandwiches": SANDWICHES_EXPORT_QUERY,
```

Comment:
forgot to say in our channel, we should do `sandwiched_swaps` too so ppl know what was sandwiched